bin/sonatype-nexus3.ts

```diff
@@ -5,10 +5,19 @@ import { SonatypeNexus3Stack } from '../lib/sonatype-nexus3-stack';
 
 const app = new cdk.App();
 new SonatypeNexus3Stack(app, 'SonatypeNexus3OnEKS', {
-    env: {
-        region: process.env.CDK_DEFAULT_REGION,
-        account: process.env.CDK_DEFAULT_ACCOUNT,
-    }
+  env: {
+    region: process.env.CDK_DEFAULT_REGION,
+    account: process.env.CDK_DEFAULT_ACCOUNT,
+  },
 });
 
-cdk.Tag.add(app, 'app', 'nexus3');
\ No newline at end of file
+new SonatypeNexus3Stack(app, 'SonatypeNexus3OnEKS2', {
```

Comment:
This will cause the app creating two Nexus deployments in two EKS clusters.

Could you pls make the `instanceType` and `createNewVpc` as context parameters?

Reply:
Also could you pls add unit tests for covering those changes?

Reply:
okay !! I try it out~~